src/citrine/resources/dataset.py

```diff
@@ -386,21 +393,42 @@ class DatasetCollection(Collection[Dataset]):
 
     Parameters
     ----------
-    project_id: UUID
-        Unique ID of the project this dataset collection belongs to.
+    team_id: UUID
+        Unique ID of the team this dataset collection belongs to.
     session: Session
         The Citrine session used to connect to the database.
 
     """
-
-    _path_template = 'projects/{project_id}/datasets'
     _individual_key = None
     _collection_key = None
     _resource = Dataset
 
-    def __init__(self, project_id: UUID, session: Session):
-        self.project_id = project_id
-        self.session: Session = session
+    def __init__(self, *args,  session: Session = None, team_id: Optional[UUID] = None, project_id: Optional[UUID] = None):
+        if len(args) > 0:
+            warn(
+                "Positional arguments are deprecated and will be removed in a future version. "
+                "Please use keyword arguments instead.",
+                DeprecationWarning
+            )
+            # Handle positional arguments for backward compatibility
+            if len(args) >= 1:
+                project_id = args[0]
+            if len(args) >= 2:
+                session = args[1]
+        self.session = session
+        self.project_id=project_id
+        if session == None:
```

Comment:
Remember to use `is None`.

Reply:
Fixed on 8d39c24072b10d099c8a0eaf580aa198cbb8efcf